_episodes/09-debugging.md

```diff
@@ -121,7 +121,7 @@ are almost never worthwhile.
 
 If it takes 20 minutes for the bug to surface,
 we can only do three experiments an hour.
-That doesn't just mean we'll get less data in more time:
+That doesn't only mean we'll get less data in more time:
```

Comment:
We need to improve this sentence -- "This doesn't only mean" is hard to parse. Simplest fix:

> This means that we'll get less date in more time and that we're more likely to be distracted...

but alternative solutions are also welcome.